pkg/cmd/docgen/diagrams.go

```diff
@@ -500,7 +500,7 @@ var specs = []stmtSpec{
 	{name: "family_def", inline: []string{"opt_name", "name_list"}},
 	{
 		name:    "grant_stmt",
-		inline:  []string{"privileges", "privilege_list", "privilege", "targets", "grantee_list", "table_pattern_list", "name_list"},
+		inline:  []string{"privileges", "privilege_list", "privilege", "targets", "table_pattern_list", "name_list"},
```

Comment:
@mjibson: is there a process for keeping this up to date? A missing type will cause a crash (it did here when regenerating the docs), but what if we forget to add a new one?